truss/tests/test_model_schema.py

```diff
@@ -0,0 +1,413 @@
+import tempfile
+from pathlib import Path
+
+import pytest
+import requests
+from truss.tests.helpers import create_truss
+from truss.tests.test_testing_utilities_for_other_tests import ensure_kill_all
+from truss.truss_handle import TrussHandle
+
+DEFAULT_CONFIG = """model_name: test-truss"""
+
+
+@pytest.mark.integration
+def test_truss_with_no_annotations():
+    truss_root = Path(__file__).parent.parent.parent.resolve()
+
+    truss_dir = truss_root / "truss" / "test_data" / "test_basic_truss"
+
+    tr = TrussHandle(truss_dir)
+
+    with ensure_kill_all():
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=True)
+        truss_server_addr = "http://localhost:8090"
+        full_url = f"{truss_server_addr}/v1/models/model:predict"
+
+        response = requests.post(full_url, json={"prompt": "value"})
+        assert response.json() == {
+            "prompt": "value",
+        }
+
+        schema_response = requests.get(f"{truss_server_addr}/v1/models/model/schema")
+        assert schema_response.status_code == 404
+
+        schema_response.json()["error"] == "No schema found"
+
+
+@pytest.mark.integration
+def test_truss_with_annotated_inputs_outputs():
+    truss_root = Path(__file__).parent.parent.parent.resolve()
+
+    truss_dir = truss_root / "examples" / "annotated_types"
+
+    tr = TrussHandle(truss_dir)
+
+    with ensure_kill_all():
+        _ = tr.docker_run(local_port=8090, detach=True, wait_for_server_ready=True)
+        truss_server_addr = "http://localhost:8090"
+        full_url = f"{truss_server_addr}/v1/models/model:predict"
```

Comment:
nit: `truss_server_addr`, `full_url`, and the schema endpoint are the same for all of these test cases. Thoughts on making them module constants at the top of the file (e.g. `TRUSS_SERVER_ADDR = "http://localhost:8090")`?